content/docs/alerting/configuration.md

```diff
@@ -63,13 +63,17 @@ global:
 
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
-  # The default SMTP smarthost used for sending emails.
+  # The default SMTP smarthost used for sending emails, including port number.
+  # Port number usually is 25 or 587 for SMTP over TLS (sometimes referred to as STARTTLS).
```

Comment:
Comma after 25

Reply:
Oxford commas are usually omitted if you only have two short items.

Reply:
This isn't an Oxford comma